src/lib/components/particles/control-change/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+export const ControlChange = ({ previous, current, string }) => (
```

Comment:
Even though it is unlikely that we will use this icon outside of the elections, I think it would be better to separate the icon into its own component, and give it a more generic name, like `ConnectedCirclesIcon` or something like that. That would give us the flexibility to use it elsewhere if we find a use for it, without necessarily including the text.

Since it looks like this particular combination of icon and text is only used in the key seat cards, I think the text should probably just live in a `KeySeatCard` component. What do you think?